src/main/java/core/basesyntax/model/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped.");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It should clearly indicate that it is an `Excavator` that has stopped working. For instance: 'Excavator has stopped working.'